packages/molly/src/routes/chat/components/chatBubble.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import Logo from './chatLogo.svelte';
+  export let expanded: boolean = false;
+
+export let expandButton = () => {
+  expanded = !expanded;
+}
+</script>
+
+<style>
+  .chat-bubble {
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  padding: 10px;
+  border-radius: 50px;
+  background-color: #323E47;
+  color: white;
+  cursor: pointer;
+  }
+</style>
```

Comment:
Could some of these styles be added to the global scope?

Reply:
I'm not able to access the global styles, not sure why